src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -90,14 +90,23 @@ namespace service_nodes
     service_nodes::participation_history<service_nodes::participation_entry> pulse_participation{};
     service_nodes::participation_history<service_nodes::timestamp_participation_entry> timestamp_participation{};
     service_nodes::participation_history<service_nodes::timesync_entry> timesync_status{};
+
+    constexpr std::array<uint16_t, 3> MIN_TIMESTAMP_VERSION{8,1,5};
+    bool check_timestamp_obligation = false;
+
     m_core.get_service_node_list().access_proof(pubkey, [&](const proof_info &proof) {
       ss_reachable             = proof.storage_server_reachable;
       timestamp                = std::max(proof.timestamp, proof.effective_timestamp);
       ips                      = proof.public_ips;
       checkpoint_participation = proof.checkpoint_participation;
-      timestamp_participation  = proof.timestamp_participation;
       pulse_participation      = proof.pulse_participation;
-      timesync_status          = proof.timesync_status;
+
+      if (proof.version >= MIN_TIMESTAMP_VERSION){
```

Comment:
i'd try using an `is_compatible_version` instead

Reply:
I think just the `>=` will be fine rather than something more generic, as this is only going to be here until the next HF and is not something that will ever change: once we are into the next hard fork we can go remove this version check code and do it unambiguously.

@darcys22 - might be worth adding a `// TODO: remove after HF17` comment here before the MIN_TIMESTAMP_VERSION.